bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -132,7 +132,7 @@ async def apply_infraction(
         infr_type = infraction["type"]
         icon = _utils.INFRACTION_ICONS[infr_type][0]
         reason = infraction["reason"]
-        expiry = time.format_infraction_with_duration(infraction["expires_at"])
+        expiry = time.format_with_duration(infraction["expires_at"])
```

Comment:
I'm not entirely sure why, or if its the same issue as the other, but this is erroring even when I give an expiry date 
```
bot_1       | 2021-09-02 20:14:11 | bot.exts.backend.error_handler | ERROR | Error executing command invoked by Chrisjl#2655: &mute 196664644113268736 7d
bot_1       | Traceback (most recent call last):
bot_1       |   File "/usr/local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
bot_1       |     ret = await coro(*args, **kwargs)
bot_1       |   File "/bot/bot/exts/moderation/infraction/infractions.py", line 159, in tempmute
bot_1       |     await self.apply_mute(ctx, user, reason, expires_at=duration)
bot_1       |   File "/bot/bot/exts/moderation/infraction/infractions.py", line 312, in apply_mute
bot_1       |     await self.apply_infraction(ctx, infraction, user, action())
bot_1       |   File "/bot/bot/exts/moderation/infraction/_scheduler.py", line 135, in apply_infraction
bot_1       |     expiry = time.format_with_duration(infraction["expires_at"])
bot_1       |   File "/bot/bot/utils/time.py", line 303, in format_with_duration
bot_1       |     duration = humanize_delta(timestamp, other_timestamp, max_units=max_units)
bot_1       |   File "/bot/bot/utils/time.py", line 196, in humanize_delta
bot_1       |     start = arrow.get(args[1]) if len(args) == 2 else arrow.utcnow()
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/api.py", line 80, in get
bot_1       |     return _factory.get(*args, **kwargs)
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/factory.py", line 228, in get
bot_1       |     raise TypeError("Cannot parse argument of type None.")
bot_1       | TypeError: Cannot parse argument of type None.
bot_1       | /usr/local/lib/python3.9/asyncio/events.py:80: RuntimeWarning: coroutine 'Infractions.apply_mute.<locals>.action' was never awaited
bot_1       |   self._context.run(self._callback, *self._args)
bot_1       | RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```